packages/imperative/src/Imperative.ts

```diff
@@ -14,51 +14,54 @@
  * require("@zowe/imperative") e.g. const imperative =  require("@zowe/imperative");
  */
 import { PerfTiming } from "@zowe/perf-timing";
-import { Logger, LoggerConfigBuilder } from "../../logger";
+import { Logger} from "../../logger/src/Logger";
+import { LoggerConfigBuilder } from "../../logger/src/LoggerConfigBuilder";
 import { IImperativeConfig } from "./doc/IImperativeConfig";
 import * as yargs from "yargs";
 import { ConfigurationLoader } from "./ConfigurationLoader";
 import { ConfigurationValidator } from "./ConfigurationValidator";
 import { ImperativeApi } from "./api/ImperativeApi";
 import { IImperativeApi } from "./api/doc/IImperativeApi";
-import { Constants } from "../../constants";
-import { ImperativeConfig, TextUtils } from "../../utilities";
-import { ImperativeReject } from "../../interfaces";
+import { Constants } from "../../constants/src/Constants";
+import { TextUtils } from "../../utilities/src/TextUtils";
+import { ImperativeConfig } from "../../utilities/src/ImperativeConfig";
+import { ImperativeReject } from "../../interfaces/src/types/ImperativeReject";
 import { LoggingConfigurer } from "./LoggingConfigurer";
 import { ImperativeError } from "../../error";
 import { PluginManagementFacility } from "./plugins/PluginManagementFacility";
 // import { ConfigManagementFacility } from "./config/ConfigManagementFacility";
-import {
-    AbstractCommandYargs,
-    CliProfileManager,
-    CommandPreparer,
-    CommandYargs,
-    ICommandDefinition,
-    ICommandProfileTypeConfiguration,
-    ICommandResponseParms,
-    IHelpGenerator,
-    IHelpGeneratorFactory,
-    IHelpGeneratorParms,
-    IYargsResponse,
-    WebHelpManager,
-    YargsConfigurer,
-    YargsDefiner
-} from "../../cmd";
-import { ProfileUtils, IProfileTypeConfiguration } from "../../profiles";
+
+import { AbstractCommandYargs } from "../../cmd/src/yargs/AbstractCommandYargs";
+import { CliProfileManager } from "../../cmd/src/profiles/CliProfileManager";
+import { CommandPreparer } from "../../cmd/src/CommandPreparer";
+import { CommandYargs } from "../../cmd/src/yargs/CommandYargs";
+import { ICommandDefinition } from "../../cmd/src/doc/ICommandDefinition";
+import { ICommandProfileTypeConfiguration } from "../../cmd/src/doc/profiles/definition/ICommandProfileTypeConfiguration";
+import { ICommandResponseParms } from "../../cmd/src/doc/response/parms/ICommandResponseParms";
+import { IHelpGenerator } from "../../cmd/src/help/doc/IHelpGenerator";
+import { IHelpGeneratorFactory } from "../../cmd/src/help/doc/IHelpGeneratorFactory";
+import { IHelpGeneratorParms } from "../../cmd/src/help/doc/IHelpGeneratorParms";
+import { IYargsResponse } from "../../cmd/src/yargs/doc/IYargsResponse";
+import { WebHelpManager } from "../../cmd/src/help/WebHelpManager";
+import { YargsConfigurer } from "../../cmd/src/yargs/YargsConfigurer";
+import { YargsDefiner } from "../../cmd/src/yargs/YargsDefiner";
```

Comment:
Please excuse my import fixing spree.
I was trying to debug a circular dependency.
This lead me to deprecate an Imperative function (`envVariablePrefix`)